unlock-app/src/__tests__/components/creator/CreatorLockForm.test.js

```diff
@@ -159,6 +163,54 @@ describe('CreatorLockForm', () => {
     })
   })
 
+  describe('currency', () => {
+    it('should default to Ether', () => {
+      expect.assertions(1)
+      const wrapper = makeLockForm()
+      expect(
+        wrapper.getByText(`Use ${config.ERC20Contract.name}`)
```

Comment:
maybe add a comment that this is asserting on the link to change the currency. I stared at this for a solid minute trying to figure out why this was a valid test :)_